DATA/production/o2dpg_qc_postproc_workflow.py

```diff
@@ -78,6 +78,7 @@ def add_QC_postprocessing(taskName, qcConfigPath, needs, runSpecific, periodSpec
 
   ## The list of QC Post-processing workflows, add the new ones below
   add_QC_postprocessing('example', 'json://${O2DPG_ROOT}/DATA/production/qc-postproc-async/example.json', needs=[], runSpecific=False, periodSpecific=False, passSpecific=True)
+  add_QC_postprocessing('EMC', 'json://${O2DPG_ROOT}/DATA/production/qc-postproc-async/emc.json', needs=[], runSpecific=False, periodSpecific=False, passSpecific=True)
```

Comment:
FYI, this means that the trends will be done for all the runs and periods which have the specific pass, e.g. `apass2`.

Reply:
Ideally we need the trending for each period and pass separately. How do we do this?

Reply:
OK, thanks, will modify.